examples/src/app/jsx.mjs

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react';
+export const jsx = React.createElement;
+export const fragment = (...args) => jsx(React.Fragment, null, ...args);
+import { createRoot } from "react-dom/client";
+
+/**
+ * @param {Function} fn - todo
```

Comment:
this could do with a small update ;)

Also some two line description of what the file is for please.

Reply:
Thank you! Fixed all `- todo` and removed these two functions (I just worked things out in the beginning with them).